src/common/sum_tree.jl

```diff
@@ -131,6 +131,24 @@ function Base.empty!(t::SumTree)
     t
 end
 
+function correct_priority(t::SumTree, leaf_ind)
```

Comment:
Can you add a docstring to this function describing what it does? (Additionally, I'm not clear on the naming, why it's called `correct_priority`...is there a more descriptive name which we could use)

Reply:
35ac46f